src/main/java/seedu/address/logic/commands/csvcommand/LoadCommand.java

```diff
@@ -0,0 +1,173 @@
+package seedu.address.logic.commands.csvcommand;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FILE_NAME;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+import seedu.address.commons.exceptions.AlfredException;
+import seedu.address.commons.util.FileUtil;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.csvcommand.csvutil.CsvUtil;
+import seedu.address.logic.commands.csvcommand.csvutil.ErrorTracker;
+import seedu.address.logic.commands.csvcommand.csvutil.ErrorTracker.Error;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.entity.Entity;
+import seedu.address.model.entity.Mentor;
+import seedu.address.model.entity.Participant;
+import seedu.address.model.entity.Team;
+
+/**
+ * Supports bulk registration via a CSV file.
+ * This command aims to facilitate registration of entities onto Alfred.
+ */
+public class LoadCommand extends Command {
+
+    public static final String COMMAND_WORD = "load";
+
+    public static final String MESSAGE_SUCCESS = "Successfully loaded CSV file into Alfred";
+    public static final String MESSAGE_PARTIAL_SUCCESS = "Following line(s) were unable to be loaded into Alfred\n"
+             + "Possible reasons include incorrect formatting or adding of duplicate Entity:";
+    public static final String MESSAGE_FILE_NOT_FOUND = "File not found at %s"; // %s -> this.csvFileName
+    public static final String MESSAGE_IO_EXCEPTION = "An IOException was caught: %s"; // %s -> exception message
```

Comment:
Is this message going to be shown to the user? If so I suggest dumbing it down because the user may not know what am IOException is. If not, then this is fine.